ntoskrnl/ke/amd64/zeropage.S

```diff
@@ -0,0 +1,29 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Fast zeroing of pages
+ * COPYRIGHT:   Copyright 2021 Jérôme Gardou <jerome.gardou@reactos.org>
+ */
+
+#include <asm.inc>
+
+/* FUNCTIONS ****************************************************************/
+.code
+
+PUBLIC KeZeroPages
+KeZeroPages:
+    pxor xmm0, xmm0
```

Comment:
Might want a `.ENDPROLOG` at least?

Reply:
your branch was nice, I wonder why it didn't make into PR state 🤔

Reply:
doh, forgot that.